rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/responder.hpp

```diff
@@ -0,0 +1,121 @@
+/* Copyright 2015 Open Source Robotics Foundation, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __rosidl_typesupport_opensplice_cpp__responder__h__
+#define __rosidl_typesupport_opensplice_cpp__responder__h__
+
+#include <ccpp_dds_dcps.h>
+#include <u_entity.h>
+
+#include <rosidl_typesupport_opensplice_cpp/message_type_support.h>
+#include <rosidl_typesupport_opensplice_cpp/service_type_support.h>
+
+namespace rosidl_typesupport_opensplice_cpp
+{
+
+template<typename RequestT, typename ResponseT>
+class Responder
+{
+  public:
+
+    Responder(DDS::DomainParticipant_ptr participant, const char * service_name,
+      const char * service_type_name) : participant_(participant),
+      service_name_(service_name), service_type_name_(service_type_name) {
+      // Create request Publisher and DataWriter
```

Comment:
Can you put the `{` on a new line aligned with the start of the method declaration? I find it hard to look at this and see where the declaration ends and the definition begins.